XpringKit/ILP/IlpClient.swift

```diff
@@ -0,0 +1,44 @@
+/// A client that can get balances and send ILP payments on a Xpring connector.
+public class IlpClient {
+
+    private let decoratedClient: IlpClientDecorator
+
+    /// Initialize a new client with a configured URL
+    ///
+    /// - Parameters:
+    ///     - grpcUrl : The gRPC URL exposed by Hermes
+    public init(grpcUrl: String) {
+        decoratedClient = DefaultIlpClient(grpcUrl: grpcUrl)
+    }
+
+    /// Get the balance of the specified account on the connector.
+    ///
+    /// - Parameters:
+    ///     -  accountId The account ID to get the balance for.
+    ///     -  bearerToken Authentication bearer token.
+    /// - Returns: A Org_Interledger_Stream_Proto_GetBalanceResponse with balance information of the specified account
+    /// - Throws: An error If the given inputs were invalid, the account doesn't exist, or authentication failed.
+    public func getBalance(for accountId: String, bearerToken: String) throws -> Org_Interledger_Stream_Proto_GetBalanceResponse {
+        return try decoratedClient.getBalance(for: accountId, bearerToken: bearerToken)
+    }
+
+    /// Send a payment from the given accountId to the destinationPaymentPointer payment pointer
+    /// - Parameters:
+    ///     -  amount : Amount to send
+    ///     -  paymentPointer : payment pointer of the receiver
+    ///     -  senderAccountId : accountId of the sender
+    ///     -  bearerToken : auth token of the sender
+    /// - Returns: A Org_Interledger_Stream_Proto_SendPaymentResponse with details about the payment. Note that this method will not
+    ///          necessarily throw an exception if the payment failed. Payment status can be checked in
+    ///          {@link SendPaymentResponse#getSuccessfulPayment()}
+    /// - Throws: An error If the given inputs were invalid.
+    public func sendPayment(amount: UInt64,
```

Comment:
Optional: 
Swift lets you name parameters so they are nicer. So you could call this method:
```
public func send(    // you can drop amount, the first parameter reads as this noun
   _ amount: UInt64, // underscore means you don't need to give it a name at the callsite
  to paymentPointer: String, // you give a `to` at call site
  from senderAccountId: String, // you give a `from` at a call site
  withAuthorization bearerToken: String)
```

Essentially this follows the pattern of:
<external name for caller> <name of variable in implementation>: <type>

So you can change the header without changing the implementation. Then to call this method it would read as:
```
send(amount, to: paymentPointer, from: senderAccountId, withAuthorization: bearerToken)
```

This makes your code more readable and flow a bit better than just repeating `<argName>: <varName>`

Generally accepted best practice in Swift API usage: 
https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage

Reply:
I like this swift thing a lot